src/main/java/com/contrastsecurity/sdk/scan/ProjectClient.java

```diff
@@ -0,0 +1,45 @@
+package com.contrastsecurity.sdk.scan;
+
+import com.contrastsecurity.exceptions.HttpResponseException;
+import com.contrastsecurity.exceptions.ResourceNotFoundException;
+import com.contrastsecurity.exceptions.ServerResponseException;
+import com.contrastsecurity.exceptions.UnauthorizedException;
+import java.io.IOException;
+import java.util.Optional;
+
+/**
+ * Internal, low-level client for making requests for project resources.
+ *
+ * @apiNote every request to the Contrast API requires an organization ID. The organization ID
+ *     parameter has been left off of the method signatures in this class, because providing it for
+ *     every request is tedious. Instead, it is expected that implementations will provide an
+ *     organization ID.
+ */
+interface ProjectClient {
+
+  /**
+   * Creates a new Scan project.
+   *
+   * @param create new project request
+   * @return the new {@link Project}
```

Comment:
If i was being pedantic I would say `ProjectInner` is not of type `Project` technically - though they are functionally equivalent.

Reply:
This is a copy paste error, I'll fix it.

Reply:
This mistake is repeated in all of the clients 🤦